pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/RepoManager.java

```diff
@@ -382,4 +400,60 @@ private void addExtraFiles(File m2Repo) {
                     strictLicenseCheck);
         }
     }
+
+    private void generateOfflinerManifest() {
+        log.info("Will generate the offliner");
+        RepositoryData result = new RepositoryData();
+        List<ArtifactWrapper> artifactsToListRaw = new ArrayList<>();
+        for (PncBuild build : builds.values()) {
+            artifactsToListRaw.addAll(build.getBuiltArtifacts());
+            // TODO: Add filter, basing on the targetRepository.repositoryType, when
+            // https://projects.engineering.redhat.com/browse/NCL-6079 is done
+            buildInfoCollector.addDependencies(build, "");
+            artifactsToListRaw.addAll(build.getDependencyArtifacts());
+            log.info("Dependencies for build " + build.getId() + ": " + build.getDependencyArtifacts().size());
+        }
+        log.info("Number of collected artifacts for offliner: " + artifactsToListRaw.size());
```

Comment:
```suggestion
        log.info("Number of collected artifacts for Offline manifest: " + artifactsToListRaw.size());
```

Reply:
Use `{}` in log statements, not `+`.

Reply:
Those statements were introduced for development purposes, so I will remove them.

Reply:
I think you can leave as `log.debug()` if you want.